bundles/org.openhab.binding.easee/src/main/resources/OH-INF/thing/things.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="easee"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="site">
+		<label>Easee Site</label>
+		<description>Cloud connection to an Easee site.</description>
+		<config-description-ref uri="thing-type:easee:site"/>
+	</bridge-type>
+	<thing-type id="mastercharger">
+		<label>Easee Master Charger</label>
+		<description>Cloud connected Easee Master Charger which also controls the circuit.</description>
+		<channel-groups>
+			<channel-group typeId="charger-state" id="state"/>
+			<channel-group typeId="charger-config" id="config"/>
+			<channel-group typeId="charger-commands" id="commands"/>
+			<channel-group typeId="charger-latest_session" id="latest_session"/>
```

Comment:
lowerCamelCase is preferred by naming convention, i.e.:
```suggestion
			<channel-group typeId="charger-latest_session" id="latestSession"/>
```

This would also be consistent with naming of channels.

Reply:
fixed